packages/core/src/components/TextField/TextField.tsx

```diff
@@ -100,6 +101,8 @@ export interface TextFieldProps extends VibeComponentProps {
    * When true, component is controlled by an external state
    */
   controlled?: boolean;
+  iconTooltipContent?: string;
+  secondaryTooltipContent?: string;
```

Comment:
those are maybe problematic (the usage of them)
does the tooltip necessarily needs to be empty if not passed this prop? we should be backwards compatible and have the tooltip show the label, no?

Reply:
The tooltip was only added now, it didn't exist in this component before, we should add it to display the label?

Reply:
but the behavior before existed

we do have tooltips today, aren't we?

Reply:
clarifyied on DM, didn't recalled we did not had tooltips up until today on TextField 🫠